requirements.txt

```diff
@@ -0,0 +1,9 @@
+ecapture==2.0.2
+pyjokes==0.6.0
+pyttsx3==2.90
+SpeechRecognition==3.9.0
+twilio==7.16.0
+wikipedia==1.4.0
+winshell==0.6
+wolframalpha==5.0.0
+PyAudio==0.2.13
```

Comment:
Check the PyAudio version. if it breaks the existing functionality because I was using PyAudio==0.2.12 in my system. Do let me know if PyAudio==0.2.13 is working without any discrepancies.